.size-snapshot.json

```diff
@@ -1,25 +1,25 @@
 {
   "dist/react-beautiful-dnd.js": {
-    "bundled": 357308,
-    "minified": 138768,
-    "gzipped": 40712
+    "bundled": 393322,
+    "minified": 147989,
+    "gzipped": 41340
   },
   "dist/react-beautiful-dnd.min.js": {
-    "bundled": 303784,
-    "minified": 113686,
-    "gzipped": 32910
+    "bundled": 327193,
+    "minified": 117669,
+    "gzipped": 33593
```

Comment:
This was suprising

Reply:
I was expecting moving away from classes would be a drop, but it has gone the other way